torch/testing/_internal/common_methods_invocations.py

```diff
@@ -68,6 +76,28 @@ def get_inplace(self):
         """
         return self.inplace_variant
 
+    # Returns True if the test should be skipped and False otherwise
+    def should_skip(self, test_name):
+        if self.tests_to_skip is None:
```

Comment:
If `tests_to_skip` was non-optional and an empty list you wouldn't need this special case.

Reply:
Sure